diff --git a/scripts/setupvars/setupvars.sh b/scripts/setupvars/setupvars.sh
index 451c51fab..cca560134 100755
--- a/scripts/setupvars/setupvars.sh
+++ b/scripts/setupvars/setupvars.sh
@@ -113,7 +113,7 @@ fi
 
 PYTHON_VERSION_MAJOR="3"
 MIN_REQUIRED_PYTHON_VERSION_MINOR="6"
-MAX_SUPPORTED_PYTHON_VERSION_MINOR="9"
+MAX_SUPPORTED_PYTHON_VERSION_MINOR="10"
 
 if  [ "$PYTHON_VERSION_MAJOR" != "$python_version_major" ] ||
     [ "$python_version_minor" -lt "$MIN_REQUIRED_PYTHON_VERSION_MINOR" ] ||
