--- setupvars.sh.orig	2021-07-10 10:56:00.000000000 +0900
+++ setupvars.sh	2021-07-11 13:20:28.000000000 +0900
@@ -29,6 +29,13 @@
 shift
 done
 
+if test -d /opt/intel/openvino/python/python3.8; then
+    python_version=3.8
+fi
+if test -d /opt/intel/openvino/python/python3.9; then
+    python_version=3.9
+fi
+
 if [ -e "$INSTALLDIR/deployment_tools/inference_engine" ]; then
     export InferenceEngine_DIR=$INTEL_OPENVINO_DIR/deployment_tools/inference_engine/share
     system_type=$(ls "$INTEL_OPENVINO_DIR/deployment_tools/inference_engine/lib/")
@@ -70,9 +77,10 @@
     if [ -f "$INSTALLDIR/opencv/setupvars.sh" ]; then
         source "$INSTALLDIR/opencv/setupvars.sh"
     else
-        export OpenCV_DIR="$INSTALLDIR/opencv/share/OpenCV"
+        export OpenCV_DIR="$INSTALLDIR/opencv/lib/cmake/opencv4"
         export LD_LIBRARY_PATH="$INSTALLDIR/opencv/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
         export LD_LIBRARY_PATH="$INSTALLDIR/opencv/share/OpenCV/3rdparty/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
+        export PYTHONPATH="$PYTHONPATH:$INSTALLDIR/opencv/lib/python$python_version/site-packages"
     fi
 fi
 
