diff --git a/scripts/setupvars/setupvars.sh b/scripts/setupvars/setupvars.sh
index e7e89c8677..2fe5a7c894 100755
--- a/scripts/setupvars/setupvars.sh
+++ b/scripts/setupvars/setupvars.sh
@@ -9,7 +9,7 @@ abs_path () {
     echo "$(cd "$directory" || exit; pwd -P)";
 }
 
-SCRIPT_DIR="$(abs_path "${BASH_SOURCE[0]}")" >/dev/null 2>&1
+SCRIPT_DIR=`dirname $0`
 INSTALLDIR="${SCRIPT_DIR}"
 export INTEL_OPENVINO_DIR="$INSTALLDIR"
 
