diff --git a/scripts/setupvars/setupvars.sh b/scripts/setupvars/setupvars.sh
index ab186881d..992301bb1 100755
--- a/scripts/setupvars/setupvars.sh
+++ b/scripts/setupvars/setupvars.sh
@@ -72,11 +72,14 @@ if [ -z "$python_version" ]; then
     python_version=$(python3 -c 'import sys; print(str(sys.version_info[0])+"."+str(sys.version_info[1]))')
 fi
 
-version_arr=(${python_version//./ })
-if [ "${#version_arr[@]}" -ge "2" ]; then
-    python_version_major=${version_arr[0]}
-    python_version_minor=${version_arr[1]}
-fi
+# version_arr=(${python_version//./ })
+# if [ "${#version_arr[@]}" -ge "2" ]; then
+#     python_version_major=${version_arr[0]}
+#     python_version_minor=${version_arr[1]}
+# fi
+
+python_version_major=$(python3 -c 'import sys; print(str(sys.version_info[0]))')
+python_version_minor=$(python3 -c 'import sys; print(str(sys.version_info[1]))')
 
 PYTHON_VERSION_MAJOR="3"
 MIN_REQUIRED_PYTHON_VERSION_MINOR="6"
